app/presenters/licences/view-licence-returns.presenter.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+/**
+ * Formats data for common licence data `/licences/{id}` page's
+ * @module ViewLicenceReturnsPresenter
+ */
+
+const { formatLongDate } = require('../base.presenter')
+
+/**
+ * Formats data for common licence data `/licences/{id}` page's
+ *
+ * @returns {Object} The data formatted for the view template
+ */
+function go (returnsData) {
+  const returns = _formatReturnToTableRow(returnsData.returns)
+
+  return {
+    activeTab: 'returns',
+    returnsUrl: 'return/internal',
+    returns
+  }
+}
+
+function _formatPurpose (purpose) {
+  const [firstPurpose] = purpose
+  return firstPurpose.alias ? firstPurpose.alias : firstPurpose.tertiary.description
+}
+
+function _formatStatus (status) {
+  if (status === 'completed') return 'COMPLETE'
+  if (status === 'due') return 'OVERDUE'
+  return 'NO STATUS'
+}
+
+function _formatReturnToTableRow (returns) {
+  return returns.map(r => {
```

Comment:
SonarCloud and us - we always use brackets even when there is only 1 arg

```suggestion
  return returns.map((r) => {
```